packages/sources/vue/src/components/overlays/VtmnToast/VtmnToast.vue

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+import '@vtmn/css-toast/dist/index-with-vars.css';
+import { reactive, computed, defineComponent, PropType } from 'vue';
+import { VtmnButton } from '../../index';
+
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnToast',
+  inheritAttrs: false,
+  components: { VtmnButton },
+  props: {
+    withIcon: {
+      type: Boolean,
+      default: false,
+    },
+    content: {
+      type: String,
+      required: true,
+    },
+    withCloseButton: {
+      type: Boolean,
+      default: false,
+    },
+    timeout: {
+      type: Number,
+      default: 8000,
+    },
+    closeCallBack: {
+      type: Function as PropType<Function>,
+      required: true,
+    },
+  },
+  setup(props) {
+    props = reactive(props);
+
+    const handleClose = () => {
+      return props.closeCallBack;
+    };
+
+    return {
+      classes: computed(() => ({
+        'vtmn-toast': true,
+        show: true,
+        [`vtmn-toast--with-icon-info`]: props.withIcon,
+      })),
+      handleClose,
+    };
+  },
+});
+</script>
+
+<template>
+  <div :class="classes" role="status" v-bind="$attrs">
+    <div class="vtmn-toast_content">{{ content }}</div>
+    <VtmnButton
+      v-if="withCloseButton"
+      iconAlone="close-line"
```

Comment:
```suggestion
      icon-alone="close-line"
```

Reply:
Oh, curious. Why? It works on our vue showcase: https://decathlon.github.io/vitamin-web/@vtmn/showcase-vue/?path=/story/components-actions-vtmnbutton--overview&args=iconAlone:apple-fill

Reply:
Yeah not changing the way it works, just to better fit with the VueJS style guide convention ! 
https://vuejs.org/style-guide/rules-strongly-recommended.html#prop-name-casing

Reply:
Oh okay! Thanks!